build.sbt

```diff
@@ -43,7 +43,7 @@ def wartSettings =
 
 lazy val commonSettings = Seq(
   organization := "org.scala-exercises",
-  version := "0.2.1-SNAPSHOT",
+  version := "0.2.2-SNAPSHOT",
```

Comment:
Just a question, can we use the same version? I think we haven't published the `0.2.1` version